class EntryNotFoundException(Exception):
    pass

class EntryAlreadyExistsException(Exception):
    pass
